app/design/adminhtml/default/default/template/ebizmarts/mailchimp/sales/order/view/monkey.phtml

```diff
@@ -16,7 +16,7 @@
                     </td>
                     <td class="value">
                         <?php if ($this->isDataAvailable()): ?>
-                            <strong><?php echo $this->quoteEscape(
+                            <strong><?php echo $this->getMailChimpHelper()->mcEscapeQuote(
```

Comment:
This can be more straight forward by calling only one function that calls the helper internally.